ds-round-trip/requirements.txt

```diff
@@ -0,0 +1,129 @@
+annotated-types==0.7.0 ; python_version >= "3.10" and python_version < "4.0"
+anyio==4.4.0 ; python_version >= "3.10" and python_version < "4.0"
+appnope==0.1.4 ; python_version >= "3.10" and python_version < "4.0" and platform_system == "Darwin"
+argon2-cffi-bindings==21.2.0 ; python_version >= "3.10" and python_version < "4.0"
+argon2-cffi==23.1.0 ; python_version >= "3.10" and python_version < "4.0"
+arrow==1.3.0 ; python_version >= "3.10" and python_version < "4.0"
+asttokens==2.4.1 ; python_version >= "3.10" and python_version < "4.0"
+async-lru==2.0.4 ; python_version >= "3.10" and python_version < "4.0"
+attrs==24.2.0 ; python_version >= "3.10" and python_version < "4.0"
+babel==2.16.0 ; python_version >= "3.10" and python_version < "4.0"
+beautifulsoup4==4.12.3 ; python_version >= "3.10" and python_version < "4.0"
```

Comment:
Are you sure that in requirements.txt is only necessary packages for this applicatioon?

Reply:
I exported it with 
```bash
poetry export --without-hashes --format=requirements.txt --output ./ds-round-trip/requirements.txt
```
It comes from Pandas, and it is true that we are not using beautifulsoup, let me specify the extra package needed when installing Pandas for reducing the size of dependencies.

Reply:
![image](https://github.com/user-attachments/assets/e1eca3dc-f9b9-49ab-be31-1c137c90fa3f)

Looks like Jupyter notebook also requires it, I don't think I can lower the dependency size further...